apps/settings/test/unit/panels/improve_browser_os/panel_test.js

```diff
@@ -63,7 +63,8 @@ suite('Improve browser os panel > ', function() {
   suite('share performance data toggle', function() {
     test('the toggle should be enabled when not dog fooding', function(done) {
       MockSettingsCache._settings = {
-        'debug.performance_data.dogfooding': null
+        'debug.performance_data.dogfooding': null,
+        'metrics.selectedMetrics.level': 'Basic'
       };
       panel.init(document.body).then(() => {
         var toggle = document.querySelector(
```

Comment:
And current test cases are not valid anymore, 

Do 

```
panel.init(document.body);
var radioA = document.querySelector('#metrics-basic');
check...
```

instead. It solves panel.init fail